util/generic/sync_map.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package lightning
```

Comment:
We can call it ```util/map```. We have ```util/set```. it will be added some generic version of ```set``` in the future.